core/src/test/scala/com/spotify/featran/transformers/HashNHotEncoderSpec.scala

```diff
@@ -16,9 +16,9 @@
  */
 
 package com.spotify.featran.transformers
-
 import com.twitter.algebird.HyperLogLogMonoid
 import org.scalacheck._
+import scala.math.ceil
```

Comment:
blank line before this according to default IntelliJ format. Or use "optimize import" to fix automatically.